bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/siteadmin/aem61/calendar/YearMonthPicker.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 Cognifide Ltd..
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.qa.bb.aem.touch.siteadmin.aem61.calendar;
+
+import com.cognifide.qa.bb.constants.Timeouts;
+import com.cognifide.qa.bb.provider.selenium.BobcatWait;
+import com.cognifide.qa.bb.qualifier.Global;
+import com.cognifide.qa.bb.qualifier.PageObject;
+import com.google.inject.Inject;
+
+import java.time.Month;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+@PageObject
+public class YearMonthPicker {
+
+  @Inject
+  private BobcatWait wait;
+
+  @FindBy(css = ".coral-Heading")
+  private WebElement yearMonthHeader;
+
+  @FindBy(css = ".coral-DatePicker-nextMonth")
+  private WebElement nextMonthButton;
+
+  void chooseMonthAndYear(int year, Month month) {
+    YearAndMonth yearAndMonth = YearAndMonth.of(yearMonthHeader.getText());
+      while(yearAndMonth.getYear() != year) {
+          nextMonthButton.click();
+          yearAndMonth = YearAndMonth.of(yearMonthHeader.getText());
+      }
+    while (!yearAndMonth.getMonth().equals(month)) {
+        nextMonthButton.click();
+        yearAndMonth = YearAndMonth.of(yearMonthHeader.getText());
+    }
+  }
+
+  private static class YearAndMonth {
```

Comment:
maybe it would be better to have methods:
getMonthFromHeader()
getYearFromHeader()
isDesiredDateSelected()
and remove static class